pallets/bridge/src/mock.rs

```diff
@@ -520,70 +594,8 @@ where
 		asset_id,
 		pre_image,
 		anchor_id,
-		(proofs, static_hashes, doc_root),
+		(proofs, doc_root, static_hashes),
 		nft_data,
 		registry_info,
 	)
 }
-
-// Return dummy proofs data useful for testing.
-//
-// This function returns proofs, static hashes, and document root.
-pub fn proofs_data<T: frame_system::Config>(
```

Comment:
are we not using these utils anymore ?

Reply:
I've already cleaned it up in a later commit.